cms/templates/js/course-instructor-details.underscore

```diff
@@ -0,0 +1,38 @@
+  <div class="field text field-course-instructor-name">
+  	<label for="course-instructor-name-<%- index %>"><%= gettext("Name") %></label>
```

Comment:
@cahrens I was done the same as you suggested but labels are not getting blue highlights as shown in image below so that is why I followed that pattern.

<img width="936" alt="screen shot 2016-04-27 at 6 11 22 pm" src="https://cloud.githubusercontent.com/assets/7334669/14853092/85f1443e-0ca3-11e6-9d28-06ab06afe238.png">